src/common/FileSystem.cpp

```diff
@@ -192,7 +192,7 @@ inline int my_open(Str::StringRef path, openMode_t mode)
 	int fd = _open_osfhandle(reinterpret_cast<intptr_t>(h), modes[mode_] | O_BINARY | O_NOINHERIT);
 	if (fd == -1)
 		CloseHandle(h);
-#elif defined(__APPLE__)
+#elif defined(__FreeBSD__) || defined(__APPLE__)
 	// O_CLOEXEC is supported from 10.7 onwards
```

Comment:
Comment should be clarified as to which OS, or deleted

Reply:
Comment clarified.